internal/resources/dataprotection/resource_enable_data_protection.go

```diff
@@ -64,6 +66,16 @@ func resourceEnableDataProtectionRead(ctx context.Context, data *schema.Resource
 	config := m.(authctx.TanzuContext)
 
 	err := populateDataFromServer(ctx, config, scopedFullnameData, data)
+
+	if scopedFullnameData.Scope == scope.ClusterScope {
+		metaData := data.Get(common.MetaKey).([]interface{})[0].(map[string]interface{})
+		annotations := metaData[common.AnnotationsKey].(map[string]interface{})
+
+		if _, ok := annotations[commonscope.BatchUIDAnnotationKey]; ok {
+			_ = schema.RemoveFromState(data, m)
```

Comment:
return after this line?

Reply:
We are anyway returning [here](https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/asagarwal/handle-tf-refresh/internal/resources/dataprotection/resource_enable_data_protection.go#L95)

Reply:
ok ideally unnecessarily shouldn't fall through to next if stmt but not big deal. feel free to resolve.